client/js/compute.js

```diff
@@ -719,6 +721,13 @@ const compute_ops = [
     sample: 'var a = colorCLuminance ${x}',
     call: function(v, x, y) { return v = calcLuminance(x); },
     hash: 'f4284956510e3fe59c2babed665b544d'
+  },
+  {
+    name: 'colorCreateHue',
+    desc: 'returns a visually distinct, random hex color using linear interpolation based on all active player colors',
+    sample: 'var a = colorCreateHue',
+    call: function(v, x, y) { return v = calcHue(activeColors); },
```

Comment:
How about we don't hardcode the array of colors to use and instead add a variable `activePlayerColors` containing the colors of active players, then this could be called as `"var a = colorCreateHue ${activePlayerColors}"` or game designers can choose a different array of colors as input.